indico/modules/designer/models/templates.py

```diff
@@ -64,9 +65,8 @@ class DesignerTemplate(db.Model):
         JSON,
         nullable=False
     )
-    background_image_id = db.Column(
```

Comment:
But even with two-sided templates storing it in a column would be much cleaner - like this we can have a proper sqlalchemy relationship and foreign keys while in JSON it's just a number that's meaningless to postgres (and thus has no enforced consistency)